osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -101,6 +106,10 @@ private void load()
             });
 
             loadNewPlayer();
+
+            this.notificationOverlay = notificationOverlay;
```

Comment:
Can be resolved with the `Resolved` attribute instead since it's usage is out of the BDL
```csharp
[Resolved]
private NotificationOverlay notificationOverlay { get; set; }
```